modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/global_settings.jspf

```diff
@@ -119,8 +119,15 @@
 			Liferay.Util.SessionStorage.TYPES.NECESSARY
 		);
 
-		if (toggleSwitch) {
-			document.getElementById(toggleSwitch).focus();
+		var element = document.getElementById(toggleSwitch);
+
+		if (element) {
+			setTimeout(function () {
```

Comment:
Don't use setTimout. Try listening for the `formReady` event, which is fired when all fields in the form are loaded. That should ensure the toggle is rendered before we scroll it into view. 

```
Liferay.once('<portlet:namespace />formReady', () => {
    // scroll element into view
});
```